Detectors/TRD/base/src/TrackletTransformer.cxx

```diff
@@ -156,3 +156,21 @@ CalibratedTracklet TrackletTransformer::transformTracklet(Tracklet64 tracklet)
 
   return calibratedTracklet;
 }
+
+double TrackletTransformer::getTimebin(double x)
+{
+  // calibration parameters need to be extracted from OCDB in the future
+  double vDrift = 1.5625; // in cm/us
+  double t0 = 4.0;        // time (in timebins) of start of drift region
+
+  double timebin;
+  if (x < -0.35) {
```

Comment:
Could you maybe add a comment what this function is supposed to do and replace the magic numbers with geometric constants?